website/package.json

```diff
@@ -5,20 +5,26 @@
   "author": "HashiCorp",
   "dependencies": {
     "@hashicorp/mktg-global-styles": "2.1.0",
-    "@hashicorp/nextjs-scripts": "13.0.0-canary.1",
+    "@hashicorp/nextjs-scripts": "14.0.1",
     "@hashicorp/react-button": "4.0.0",
     "@hashicorp/react-case-study-slider": "3.0.3",
+    "@hashicorp/react-code-block": "3.0.3",
     "@hashicorp/react-docs-page": "10.3.2",
     "@hashicorp/react-hashi-stack-menu": "1.0.11",
     "@hashicorp/react-head": "1.1.6",
     "@hashicorp/react-hero": "4.1.0",
     "@hashicorp/react-image": "3.0.3",
     "@hashicorp/react-inline-svg": "5.0.0",
+    "@hashicorp/react-logo-grid": "3.0.1",
```

Comment:
Previously this was being pulled in indirectly via `text-and-content`. With the `text-and-content` dependency removed, this needed to be added as an explicit dependency.